en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -472,4 +473,75 @@ To create a For Each Loop:
 
 You can add logic and actions in different patterns, depending on what your flow needs to do.
 
+== Designate Error Handling 
+
+When executing a flow, you can watch for errors in connection or logic in specific steps of your flow and then designate logic to handle the errors that may occur. You can also designate steps to complete additional actions or to send notifications in the event of an error. 
+
+[[access-error-handling]]
+=== Access Error Handling 
+
+To access the Error Handler:
+
+. <<create-a-new-flow,Create a new flow>> or open an existing flow.
+. If you create a new flow:
+.. Select the connector or system to start your flow.
+.. Select and configure a trigger. 
+.. Select and configure an action.
+.. Directly before the step to which you want to add an Error Handler, click the plus button. 
+.. In the *What Should Start This Flow?* window, click *Error Handler (Beta)*.
+. If you add a step to an existing flow:
+.. Open your flow.
+.. Directly before the step to which you want to add an Error Handler, click the plus button. 
+.. In the *Next Step* window, click *Error Handler (Beta)*. +
```

Comment:
Extra at the end here?

Reply:
Yes - the OG content had a result statement but I decided it was superfluous. Good catch.